fenixedu-ist-reports/src/main/java/pt/ist/fenix/jasper/reports/JRTxThreadSubreportRunner.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * JasperReports - Free Java Reporting Library.
+ * Copyright (C) 2001 - 2009 Jaspersoft Corporation. All rights reserved.
```

Comment:
Why are we including Jaspersoft licensed source code inside the project, shouldn't this be a dependency?